legacy-sdk/whirlpool/tests/utils/init-utils.ts

```diff
@@ -226,93 +228,85 @@ export async function buildTestAquariums(
       }),
     );
 
-    const pools = await Promise.all(
-      initPoolParams.map(async (initPoolParam) => {
-        const {
-          tickSpacing,
-          mintIndices,
-          initSqrtPrice = DEFAULT_SQRT_PRICE,
-          feeTierIndex = 0,
-        } = initPoolParam;
-        const [mintOne, mintTwo] = mintIndices.map((idx) => mintKeys[idx]);
-        const [tokenMintA, tokenMintB] = PoolUtil.orderMints(
-          mintOne,
-          mintTwo,
-        ).map(AddressUtil.toPubKey);
-
-        const configKey =
-          configParams!.configInitInfo.whirlpoolsConfigKeypair.publicKey;
-        const whirlpoolPda = PDAUtil.getWhirlpool(
-          ctx.program.programId,
-          configKey,
-          tokenMintA,
-          tokenMintB,
-          tickSpacing,
-        );
+    // Initialize pools sequentially to avoid AccountBorrowFailed errors in LiteSVM
+    const pools: InitPoolParams[] = [];
+    for (const initPoolParam of initPoolParams) {
+      const {
+        tickSpacing,
+        mintIndices,
+        initSqrtPrice = DEFAULT_SQRT_PRICE,
+        feeTierIndex = 0,
+      } = initPoolParam;
+      const [mintOne, mintTwo] = mintIndices.map((idx) => mintKeys[idx]);
+      const [tokenMintA, tokenMintB] = PoolUtil.orderMints(
+        mintOne,
+        mintTwo,
+      ).map(AddressUtil.toPubKey);
+
+      const configKey =
+        configParams!.configInitInfo.whirlpoolsConfigKeypair.publicKey;
```

Comment:
i vaguely remember that we might've updated configParams to have a default so we didn't need the shebang?